agent/acs/handler/acs_handler.go

```diff
@@ -290,8 +290,10 @@ func (acsSession *session) startACSSession(client wsclient.ClientServer) error {
 		cfg.Cluster,
 		acsSession.containerInstanceARN,
 		client,
-		acsSession.state,
-		acsSession.dataClient,
+		&dockerENIHandler{
```

Comment:
What does "dockerENI" mean in this context? I'm curious why is "docker" in the name, isn't an EC2 ENI independent of docker?

Reply:
Originally the intent of the name "dockerENIHandler" was to express that this struct is used to handle ENIs for our agent, which is Docker based. It is not meant to indicate that ENIs are dependent on Docker. 

A more explicit name to get the intended point across would likely be something like "dockerAgentENIHandler". But on second thought I think adding this information to the struct name is unnecessary and redundant, since this struct which implements ENIHandler lives in the `agent/` directory, which we already know to be Docker based. As a result, I am thinking to just name the struct "eniHandler" instead. What do you think?

Reply:
sure, that sounds good to me